common/src/main/java/dev/lavalink/youtube/http/YoutubeOauth2Handler.java

```diff
@@ -289,6 +290,16 @@ public void applyToken(HttpUriRequest request) {
         }
     }
 
+    public void applyToken(HttpUriRequest request, String token) {
+        if (!Client.PLAYER_URL.equals(request.getURI().toString())) {
+            return;
+        }
+
```

Comment:
I don't think this is needed, the if from the caller already ensures this is the case